rdbms/src/main/java/io/spine/server/storage/jdbc/record/JdbcRecordStorage.java

```diff
@@ -112,10 +114,16 @@ public boolean delete(I id) {
     }
 
     @Override
-    protected Iterator<EntityRecord> readMultipleRecords(Iterable<I> ids,
-                                                         FieldMask fieldMask) {
-        Iterator<EntityRecord> records = table.readByQuery(toQuery(ids), fieldMask);
-        return records;
+    protected Iterator<@Nullable EntityRecord> readMultipleRecords(Iterable<I> ids,
+                                                                   FieldMask fieldMask) {
+        Collection<EntityRecord> records = new ArrayList<>();
+        for (I id : ids) {
+            EntityQuery<I> query = toQuery(id);
+            EntityRecord recordOrNull = readRecordOrNull(query, fieldMask);
```

Comment:
This seems rather inefficient. We should keep the number of calls to the DB as low as possible.